htdocs/core/menus/standard/eldy.lib.php

```diff
@@ -75,9 +75,15 @@ function print_eldy_menu($db, $atarget, $type_user, &$tabMenu, &$menu, $noout =
 	$menu_arr = array();
 
 	// Home
+	$landingpage = (empty($user->conf->MAIN_LANDING_PAGE) ? (!getDolGlobalString('MAIN_LANDING_PAGE') ? '' : $conf->global->MAIN_LANDING_PAGE) : $user->conf->MAIN_LANDING_PAGE);
```

Comment:
Use of conf->global->xxx or user->conf->xxx is noore allowed.

Reply:
sorry it was a copy/paste from main.inc.php :wink:

Reply:
$user->conf-> is also not allowed.
use getDolUserString